index.html

```diff
@@ -1052,7 +1058,7 @@ <h2>Method-Specific DID URL Parameters</h2>
         </p>
 
         <pre class="example nohighlight">
-did:example:21tDAKCERh95uGgKbJNHYp;service=agent;foo:bar=high
+did:example:21tDAKCERh95uGgKbJNHYp?service=agent&foo:bar=high
```

Comment:
prefer not to see foo:bar as the example here if there is _any_ better option.

Reply:
Thanks @OR13 , I think this is orthogonal to the matrix/query parameter change and should be discussed in another issue (e.g. in https://github.com/w3c/did-core/issues/36 or a new one).

Reply:
yes I agree. you can resolve this.